src/content/docs/logs/ui-data/query-syntax-logs.mdx

```diff
@@ -47,8 +47,22 @@ General query rules:
 
       <td>
         The query syntax is case sensitive for attributes values. Attribute names are always case sensitive.
+      </td>
+    </tr>
 
-        **Exception:** Wildcard searches are case insensitive for attribute values.
+    <tr>
+      <td>
+        Whitespace characters
```

Comment:
Adding a separate section about whitespace (since it's different from metacharacters -- you still need to quote stuff, but you don't need to escape them)

Reply:
Excellent!